docs/source/best-practices.md

```diff
@@ -4,6 +4,13 @@
 
 See the [contribution guidelines for leapp-repository](contributing).
 
+## Naming convention
```

Comment:
This is IMHO not a best practice. In best practices we shouldn't use the word `must`. Instead, best practices should contain stuff that's not mandatory but highly recommended.
I'd put it to the `docs/source/contributing.md` under a section `File naming convention`.

Reply:
I think this Naming convention is more confusing then helpful. 
- Why do we require do not use underscores? 
- Then requirements to the test modules we have in testing actors doc. 
- Do not use actor name in modules sounds like common sense and we don't need to write common sense derived things here, otherwise, we could never end.
- And requirement to the naming of a main module of actor actor.py is looking like mandatory and should probably stay in best-practices for actor development. 

I suggest keeping the actor.py requirement inside best-practices and this is it. The rest should be skipped.

Reply:
> I think this Naming convention is more confusing then helpful.
> 
>     * Why do we require do not use underscores?
To have the naming consistent, it's as easy as that. We didn't want one actor be named `checkonething` and the other `check_the_other_thing`. I'd not remove this one.
> 
>     * Then requirements to the test modules we have in testing actors doc.
I don't understand this bullet.
> 
>     * Do not use actor name in modules sounds like common sense and we don't need to write common sense derived things here, otherwise, we could never end.
It may seem like a common sense now because you see now tens of actors and models that follow this rule. But it was not like that from the beginning. I agree that it can be dropped now because there's already enough examples to give you the idea of naming.
> 
>     * And requirement to the naming of a main module of actor actor.py is looking like mandatory and should probably stay in best-practices for actor development.
For mandatory requirements in documentation, word `shall` is being used instead of `must`. And as I wrote, I'd like to avoid having mandatory requirements in best practices. It sounds like having word `must` in a meal recipe. This is not a best practice, it's a requirement mandated by the leapp framework.
> 
> 
> I suggest keeping the actor.py requirement inside best-practices and this is it. The rest should be skipped.

Reply:
I think the naming convention should really be part of contribution guidelines. Best practices tells, you doesn't have to do it neccessary. But in this case, we are talking about something what will be required. As well, I think that is not needed to put there any duplicated info about name of actor script `actor.py` - that's the requirement from the framework already, so no need to document that twice.

> Why do we require do not use underscores?

long story short, we use (or is it just planned?) underscores to be able to create uniq key for the actor based on the path in the repository.

Reply:
> * Then requirements to the test modules we have in testing actors doc.
I meant that how to test modules should be named already described in 
`test-actors.md`.

The requirement about the naming of `actor.py` is also here: docs/source/repository-dir-layout.md. I don't see any other reference from the leapp framework. I would keep it in contributing.md


So should I go with:
```diff
--- a/docs/source/contributing.md
+++ b/docs/source/contributing.md
@@ -15,7 +15,14 @@ Your code should follow our
 
 * Actor development: [Best practices for actor development](best-practices)
 * Actor directory layout is to be found at [Repository directory layout](repository-dir-layout) (under `actorname/`)
-- actor testing: [Testing actors](test-actors.html)
+* Actor testing: [Testing actors](test-actors.html)
+
+## File naming convention
+
+1. New folders and/or Python files shall use lowercase without underscores.
+1. Do not use _actor_ in the actor's name, neither _model/topic/tag/phase_ 
+in the model/topic/tag/phase name, as these things are implicit.
+1. The actor's main file shall be named `actor.py`.
```
?